### DISCUSSION TITLE
Use case question

### DISCUSSION THREAD

**Discussion by LobsterMan123** [psambit9791/jDSP#57] [Q&A] [answered]
Would you please tell me if by calculating the "only positive absolute values" from JDSP when doing the Fourier transform, if this is sufficient, for a physical system I have that is oscillating back and forth, to obtain the constituent frequencies of oscillations in my system?

The code I'm using to calculate the only positive absolute values is below:

`_Fourier ft = new DiscreteFourier(signal);
// OR
_Fourier ft = new FastFourier(signal);
ft.transform();
boolean onlyPositive = true;
double[] out = ft.getMagnitude(onlyPositive); `

Thanks.

LM

**Comment by psambit9791** [comment, marked as answer]
@LobsterMan123 Should be fine.
